Update alias list API docs with filtering options (#861)
zeit/docs: pages/docs/api/v1/api-docs-mdx/endpoints/aliases.mdx

<Endpoint method="GET" url="/v2/now/aliases" />

> Retrieves all of the active now aliases for the authenticating account.

Reviewer: ```suggestion
Retrieves all of the active aliases for the authenticated account.
```

#### Request Query Parameters

> The response of this API can be controlled with the following query parameters. All of them are optional.

Reviewer: ```suggestion
The response of this API can be controlled with the following optional query parameters.
```

| Key           | Description                                          |
| ------------- | ---------------------------------------------------- |
| **limit**     | Maximum number of aliases to list from a request.    |
| **from**      | Get aliases created after this JavaScript timestamp. |

> | **projectId** | Filter aliases from the given projectId.             |

Reviewer: ```suggestion
| **projectId** | Filter aliases from the given `projectId`.             |
```
